integration/substrate/call_flags.sol

```diff
@@ -0,0 +1,58 @@
+contract CallFlags {
+    uint8 roundtrips;
+
+    // See https://github.com/paritytech/substrate/blob/5ea6d95309aaccfa399c5f72e5a14a4b7c6c4ca1/frame/contracts/src/wasm/runtime.rs#L373
+    enum CallFlag { FORWARD_INPUT, CLONE_INPUT, TAIL_CALL, ALLOW_REENTRY }
```

Comment:
How about defining these flags in `import "substrate";`?

Reply:
I thought about it but decided against it. This doesn't need to be a compiler builtin; there's nothing that can't be written in plain Solidity. I'd rather provide such helper things directly in Solidity (sort of like a stdlib).